chapter_explanability/overview.md

```diff
@@ -0,0 +1,57 @@
+* be aware of the perils and pitfalls existing in the domain.
+
+## Taxonomies in model explainability
+
+Explainability in machine learning is the degree to which a human can understand the cause of a decision made by a machine learning model. As shown in Figure :numref:`xdl_intro`, explanation methods take as inputs the data and the black box model, and provide users with human-comprehensible explanations which are utilized to assist the decision-making process.
+
+![Model explanability.](../img/xdl.svg)
+:label:xdl_intro
+
+#### Inherently interpretable models vs. post-hoc explanations
+While some conventional machine learning models:cite:`Bishop.2006` such as linear regression, logistic regression, naive Bayes classifier, k-nearest neighbors, decision trees, decision set, and generalized additive models (GAMs):cite:`Hastie.Tibshirani.2017` are inherently explainable, most black box models (e.g., deep neural networks, random forests:cite:`Breiman.2001`, support vector machines:cite:`Scholkopf.Smola.2002`, XGBoost:cite:`Chen.Guestrin.2016`, etc) which are often demonstrated to be more effective in many real-world applications are not. In recent years, a variety of approaches are proposed to enhance the explainability of black box models. 
+
+#### Forms of Explanations
+Explanations come in various forms. It can not only be simple formats such as numerical feature importance values, saliency maps, readable sentences, graphs, and bar plots, but also be a bunch of representative instances or a simple model which can be used to approximate an opaque model.
+
+
+#### Global explanations vs. local explanations
+Pertaining the scope of explanations, we can classify explanations into global explanations and local explanations. Global explanation methods describe the average behavior of a black box model. A typical global explanation method is the global feature importance plot which displays how much impact each feature has on average for model predictions. On the contrary, local explanation methods are centered around the prediction of each instance and focus on explaining how a certain prediction is obtained for an individual instance. For instance, we will know which pixels make an image to be classified as a bird via local explanation methods for image classifiers.
+
+#### Model-agnostic methods vs. model specific methods
+Based on the applicability of explanation methods, we have model agnostic explanation methods and model specific explanation methods. The former is more flexible and can be applied to any black box model regardless of its structure while the latter is limited to specific model classes.
+
+
+#### Desired properties of eplanations 
+There is no standard evaluation measures on explainability but the following dimensions, which might be difficult to quantify rigorously, can be considered as guiding intuitions when developing or evaluating explanation methods:
```

Comment:
Are we going to refer to these properties in the rest of the chapter?

Reply:
I am not quite sure if we are gonna refer to them  during implementations since it is quite difficult to quantify these properties, but we can discuss them later in more detail.